pkg/runner/assets/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source ./lib.bash
+source ./env
+source ./steps.sh
+
+step_prefix="step"
+
+run() {
```

Comment:
this got moved into `step::run` in `lib.bash` any reason to copy and paste it here?

Reply:
Consolidating all the logic for executing a run into this script. lib keeping as utilities to be used in setup script.

Reply:
Yah, but I moved it into lib on purpose, so it can be used by multiple pieces of code, as an agnostic way to run steps.  Why pull it out of the library?

Reply:
Keeping in the lib.sh